setup.py

```diff
@@ -13,6 +13,19 @@ def get_files_in_folder(directory):
     return file_list
 
 
+install_requires = [
+            'numpy',
+            'shapely',
+            'PyYAML',
+            'pycollada',
+            'astar',
+            'descartes',
+            'trimesh',
+            'scipy',
+            'PyQt5',
+            'adjustText']
```

Comment:
Personal preference to use "double quotes" instead of 'single' for strings, and to put in alphabetical order so it's easier to follow and compare to other things (e.g. the output of `pip freeze`).

Also, run it through an autoformatter so there are 4 spaces. I use the default autoformatter for Python in VSCode where you can press Ctrl+K CtrL+F to format stuff.

Finally, these steps let you run the regular non-ROS test script but it's missing an install to `transforms3d` to run the ROS2 examples. So add that in too.

```suggestion
install_requires = [
    "adjustText"
    "astar",
    "descartes",
    "numpy",
    "pycollada",
    "PyQt5",
    "PyYAML",    
    "shapely",
    "scipy",
    "transforms3d",
    "trimesh"]
```